ee/management/commands/materialize_columns.py

```diff
@@ -0,0 +1,73 @@
+import logging
+
+from django.core.management.base import BaseCommand
+
+from ee.clickhouse.materialized_columns import materialize
+from ee.clickhouse.materialized_columns.analyze import logger, materialize_properties_task
+from posthog.settings import (
+    MATERIALIZE_COLUMNS_ANALYSIS_PERIOD_HOURS,
+    MATERIALIZE_COLUMNS_BACKFILL_PERIOD_DAYS,
+    MATERIALIZE_COLUMNS_MAX_AT_ONCE,
+    MATERIALIZE_COLUMNS_MINIMUM_QUERY_TIME,
+)
+
+
+class Command(BaseCommand):
+    help = "Materialize properties into columns in clickhouse"
+
+    def add_arguments(self, parser):
+        parser.add_argument("--dry-run", action="store_true", help="Print plan instead of executing it")
+
+        parser.add_argument("--property", help="Property to materialize. Skips analysis.")
+        parser.add_argument(
+            "--property-table", type=str, default="events", choices=["events", "person"], help="Table of --property"
```

Comment:
Would specifying this also mean that we wouldn't backfill? Is this desired?

We could instead pass in the property to `materialize_properties_task` and reuse the logic there.

Reply:
Resolved!